bool enable_velocity_ctrl
uint16 pwm_max 
uint32 KP
uint32 KD
